compiler/parser/parser.peg

```diff
@@ -480,8 +480,8 @@ Pool
   = "pool" _ body:PoolBody { RETURN(body) }
 
 PoolBody
-  = spec:PoolSpec at:PoolAt? over:PoolRange? order:OrderArg? {
-      RETURN(MAP("kind": "Pool", "spec": spec, "at": at, "range": over, "scan_order": order))
+  = spec:PoolSpec at:PoolAt? order:OrderArg? {
```

Comment:
To avoid breakage for users, @mccanne and I think we should keep this in the language but remove it from the documentation and examples. That probably means keeping it in the AST, removing it from the DAG, and creating a `dag.Filter` for it during semantic analysis. Happy to talk that over if you'd like.

Reply:
Ok.